### PR TITLE
Update spec to align with lab template

### LINKED ISSUE #3: Fixed Specs
It seems that the span element wasn't getting clicked in the specs, and therefore wasn't able to test if `'Thank you for clicking!'` was being displayed in the browser. To correct this, all I had to do was add `span.click();`. See line 13 below: 

screenshot 2016-03-26 04 16 56

Also, since you aren't testing the text that's displayed in the directive variable, you could condense the spec to the following: 

screenshot 2016-03-26 04 25 15

### PR DESCRIPTION
**Pull request by mendelB**
Fixes #3 and #2 to test the proper elements in the directive (Just a small change to what's being clicked, and where the test expects to see changed text)
@ipc103

### CODE REVIEW

File: spec/Index.spec.js [hunk 1]
```diff
@@ -4,12 +4,12 @@ describe('Directive Test', function() {
 	browser.get('http://localhost:8080');
 
 	it('should update the text on click', function() {
-		var directive = element(by.css('some-directive div'));
+		var directive = element(by.css('some-directive div span'));
 
 		directive.click();
```
Comment (aturkewi) [inline spec/Index.spec.js:9]: @mendelB I would update the variable `directive` on lines 7 and 9 to `span`

**Comment by aturkewi** [comment]
@ipc103 I took a look at this, and the changes look good. Just asked @mendelB  to update a variable name. 

Keeping @PeterBell in the loop

**Comment by PeterBell** [comment]
Looks good to me. Just merge it in when you're both good with it!

**Comment by ipc103** [comment]
thanks all - going to merge and update the solution to match.

**Pull request merged**